docs/quickstart_pubFlow.md

```diff
@@ -1,12 +1,13 @@
 # Quickstart: Publisher flow
 
-## This is a sample on how a publisher is going to interact with the contracts.
-### Data Publisher is going to create a new NFT contract, then from there will deploy 3 new DTs used for different licensings, and then create a different type of go to market strategy for each DT. No Roles are used in this one, since the Data Publisher is the NFTOwner, he has all permissions.
+## This is a sample on how a data publisher is going to interact with the contracts.
 
+### DataPublisher is going to create a new NFT contract, then from there will deploy 3 new DTs used for different licensings, and then create a different type of go to market strategy for each DT. No Roles are used in this one, since the Data Publisher is the NFTOwner, he has all permissions.
```

Comment:
It shouldn't be "this is how it's going to be". 

It should be "this is how it is".